app/models/admin/FeatureFlags.scala

```diff
@@ -144,3 +140,31 @@ case object BreathingSpaceIndicatorToggle extends FeatureFlagName {
     "Enable/disable calls to Breathing space"
   )
 }
+
+case object AddressJourneyEnforcerToggle extends FeatureFlagName {
+  override val name: String                = "address-journey-enforcer-toggle"
+  override val description: Option[String] = Some(
+    "Enable/disable call to for address journey"
+  )
+}
+
+case object GetPersonFromCitizenDetailsToggle extends FeatureFlagName {
+  override val name: String                = "get-person-from-citizen-details-toggle"
+  override val description: Option[String] = Some(
+    "Enable/disable call to designatory-details"
+  )
+}
+
+case object DfsDigitalFormFrontendShutteredToggle extends FeatureFlagName {
+  override val name: String                = "dfs-digital-forms-frontend-shuttered-toggle"
+  override val description: Option[String] = Some(
+    "Enable/disable indicating whether the dfs digital forms frontend is shuttered"
+  )
+}
+
+case object AddressChangeAllowedToggle extends FeatureFlagName {
```

Comment:
What's the difference with AddressJourneyEnforcerToggle?

Reply:
The logic is opposite to the other toggles above. i.e. when all is well this one is set to false while the other are set to true.

Reply:
AddressJourneyEnforcerToggle is the replacement for NpsOutageToggle in the addressJourneyEnforcer. AddressChangeAllowedToggle is the toggle added for the change link to be shown or hidden

Reply:
And When all is well this one is also set to true not false.